lib/holdem_game.js

```diff
@@ -142,6 +126,8 @@ HoldemGame.prototype.gameStart = function() {
 	var room = this;
 	var seats = room.seats;
 	var gamers = room.gamers;
+	console.log('gamestart');
+	console.log(room.deal_order);
```

Comment:
You should not keep console logs around.

Reply:
Thanks for spotting, I removed the console logs.